api/agent.go

```diff
@@ -1237,15 +1238,15 @@ func (a *Agent) updateTokenOnce(target, token string, q *WriteOptions) (*WriteMe
 	if err != nil {
 		return nil, 0, err
 	}
-	defer closeResponseBody(resp)
 
 	wm := &WriteMeta{RequestTime: rtt}
 
-	if resp.StatusCode != 200 {
-		var buf bytes.Buffer
-		io.Copy(&buf, resp.Body)
-		return wm, resp.StatusCode, fmt.Errorf("Unexpected response code: %d (%s)", resp.StatusCode, buf.Bytes())
+	_, _, err = requireOK(rtt, resp, err)
+
```

Comment:
minor: remove newline here

Reply:
hm, I can certainly do that but I was hoping that any style nits were taken care of by `gofmt`